wkt_to_dicomseg/main.ipynb

```
@@ -0,0 +1,287 @@
Y4zEltpT3PzNxhbzyXVzggSC1Jy4wE9jcz94HXJ9eG/8Meuf+wm+vPXAhKQeu426V47L23HiPtmECvSzNAZKQ56NMN+ArOvjufkDHmEQoz+89QyX8+q14z7XBys3kNdXrscex/fl5aVcXl7Ker1+4bctBw1DL+fn5/Lhhx/I+fmZfO5zn5OTk9WoBxcyMxVEhPH7uPaONQrZuEbdH8V4cROHDOPaHKxd48/7sj5b91w5PqZZt8mB8sqoTAeB8QzMs3oX200l3rtR73q4Jll7ySHLnY8az3N2ijXwRdTxymWlTpltT/Vc7vja9iEHTcjS1+9441v+9/vjtZNfYzPH9QaQxXlPN1o8/3+8++/kl6zbSVezjdH3/WSf1exBpjAMMqgQzajk48dtxbFTeMXHK10nP/7L70sX3jTjoDe4Uh2wa1KWem4v5o6gZ1Pl/fKz6UWAxboNYG2e5r/xDHtPdo3nnPP417HPa8ti3LxzLEW5YylvR7eOxLrmeg9h91x/tL1cn9fmsp3r2+1WNhvdIaqRUsNw267+bhjeMPyThuF5n9PYPAkMH4YgIQykD+y4+BiOMePXionkPl7D8LKthuENw4+hht+2Xf3d8Lvhd8NvyKbht/5u+H1suw2/Ez1WoFtE5Pz8XH7zm9/Ihx9+KFdXV/LKK6/IarWahFgLqoj4A1QTfI14ojyJcpa3OQD1znnHrQJE/fb84/TDq9e7hn/rdhb5ltkeqLAStcYDBxpFJAtyhxCyLcpFZMwGClmAFNfaICfKeYsc5znw5pWpAV0I+Xu2uS2c98YMx+baBdWUgh1/D5gZsLz+M/8AMj5n2/LasQrUBttDCGR4pSD3brebPrimlr3IdeM35MayH4b0bncE0MtMtHK7dx4fnk8wJGKMU10womLMExRsIJWNEJ6HPA5cxq5du374tx0ryMKOK++GMacneMzmDHrO1LTysgF1btczSjxDyfbB1junU2y/rK5i3vDNCRpWTt6x/X4vFxcXcn5+LrvdrpqM8CLS1dWl3LlzVx4+PJXNZiM3b96QxQJjUc41Jh+//d+PSuwc1fDxOrjO9epl6hTmx1G2brx6enauTD6/1aE6xKuXQYmMcotT3mseRNR5gdGvBjXrLiS8oS5OyglT/daBsdtYoV9z+rCUB8s5ZHz7cmFMRJapOuch8E4u5fhamdm6OROWac554PN8fe7IwfHPk6O4+3Dc0HdPjoVdsM9vmvCtDjzF3fe97Pte+v0+bWse4/RdMDFJksYmhGlrdNbPU52jjeDJ6vX/bpDhP/9GpMvnbOK3tHk4+RK2Dur3d2VSp1xxWzIZYz4gk363E1kui26PvGsfwK+OFd+I0evTuKGySN8+dlvybLycp0fD7lqdpV3JN6wS31792jfft7DEMirP1dfYMAyy2Wxks1nLfr+TT9qWaR8FNQxH/Q3DG4bndX/iMFxyDOd5jQBKwre97Pe96NNNA8nE9jbHa+abMRx1zmF4erqwvD/SMJyk3TA8O9YwfJ4afqP+ht8Nv/O6G34LybLhd8Nvrb/h9/H02IHuq6sref/99+V3v/udnJ2dyWuvvSY3btx4bMaOAU4sxrlrvfMhhMoi9euZmzxcJ/+2AHPoej5/iDeu81g58W9WjuDX21rcBnFZMWKR80JHQPvk5EQ2m03xVK5ICuLt93vZbrfTNtL8pLcNcIMP3tKcearJrabMuE4rH/TvGIOyNj6erPla+98GiPl6e02tHduXmuHgXW+TCCz/eN86ArsIdAPsGBC5v1amvH05ArkA4xDyYDoHonEtE2f+2SQLlgEH7Xnu2O3M8c0ZlXwtH+NxqwXMy/eNaDv6Xp1cX4BvyMJ7whs8scHg1eMlX3i6iNv19IIH1rafzAu35a0Plp9tg497dc7perumLB/b7VYuLi7k8vKy2GniRafNZiv37t2Te/fuTolqy+Xjy+cYER+HXd7R9G6imgNSlD4Cv5PDGadvW3zOpphzHOuk2dGH
```

What is the added value in the try/except here? Usually we keep recipes as simple as possible, and it seems simpler to let the exception be raised directly rather than printing it.